subprojects/docs/src/docs/userguide/img/taskInputsOutputs.graffle

```diff
@@ -0,0 +1,816 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I agree, although we're not expecting significant changes to the user manual in the near future.